src/clients/buildclient.ts

```diff
@@ -62,6 +67,35 @@ export class BuildClient extends BaseClient {
         }
     }
 
+    //Gets the appropriate build for TFVC repositories and returns a 'BuildBadge' for it
+    private async getTfvcBuildBadge(svc: BuildService, teamProjectId: string): Promise<BuildBadge> {
+        //Create an build that doesn't exist and use as the default
+        let emptyBuild: BuildBadge = { buildId: undefined, imageUrl: undefined };
+
+        let builds: Build[] = await svc.GetBuilds(teamProjectId);
+        if (builds.length === 0) {
+            return emptyBuild;
+        }
+
+        let matchingBuild: Build;
+        for (let idx = 0; idx < builds.length; idx++) {
+            let b: Build = builds[idx];
+            // Ignore canceled builds
+            if (b.result === BuildResult.Canceled) {
+                continue;
+            }
+            if (b.repository &&
+                b.repository.type.toLowerCase() === "tfsversioncontrol") {
+                    matchingBuild = b;
+                    break;
+            }
+        }
+        if (matchingBuild) {
+            return { buildId: matchingBuild.id, imageUrl: undefined };
```

Comment:
Is it important to set the imageUrl here? Just checking. Maybe a comment as to why we are leaving it empty?